src/Form/Form.js

```diff
@@ -24,43 +25,87 @@ const Form = (props) => {
     setNewCurrency('')
   }
 
-  const toInputUppercase = e => {
-    e.target.value = ("" + e.target.value).toUpperCase();
-  };
-  
-  return (
-    <section >
+  if(!userCurrency || !userAmount || !newCurrency) {
```

Comment:
There is probably a better way to do this if statement (without having the render return 2x), but it works and I good with that for today! :)